src/app.controller.ts

```diff
@@ -1,12 +1,27 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, Post, Body } from '@nestjs/common';
 import { AppService } from './app.service';
+import * as data from './sample-alert.json'
+import { LogMessageFormat } from 'logging-format';
 
 @Controller()
 export class AppController {
   constructor(private readonly appService: AppService) {}
 
-  @Get()
-  getHello(): string {
-    return this.appService.getHello();
+    /**
+     * Example to show converted Sample-Alerts into LogMessages.
+     */
+  @Get('/get-sample')
+  async respondToRequest(): Promise<any> {       
+    return this.appService.convertAlertToLogMessages(data);
   }
+
+  /**
+   * Endpoint for the Alert-Manager, incoming Alert will be converted and sent into the Queue.
```

Comment:
welche queue? lässt sich das genauer sagen? 

außerdem heißt ist es m.e. nicht 'into' sondern 'to'. wir senden nachrichten im allgemeinen 'an' und nicht 'in' die queue.